components/cam/src/control/camsrfexch.F90

```diff
@@ -594,9 +598,9 @@ subroutine cam_export(state,cam_out,pbuf)
          !zm will use large-scalel (CAM) value
          cam_out%zbot(i,j)  = state%zm(i,pver)
 
-         !u and v will use CRM value
-         cam_out%ubot(i,j)  = crm_u(i,j,1,1)
-         cam_out%vbot(i,j)  = crm_v(i,j,1,1)
+         ! u and v will use CRM value (must transform because of CRM orientation)
+         cam_out%ubot(i,j) = crm_u(i,j,1,1) * cos(crm_angle(i)) - crm_v(i,j,1,1) * sin(crm_angle(i))
+         cam_out%vbot(i,j) = crm_v(i,j,1,1) * cos(crm_angle(i)) + crm_u(i,j,1,1) * sin(crm_angle(i))
```

Comment:
This looks to me right, but just want to make sure that the "+" and "-" sign are correct. Can you confirm that? There must be a math transformation, but I don't know how to do it...

Reply:
Good catch, looks like the signs need to be reversed. The formula used in crm_physics are correct. The part where CRM winds are rotated back to the GCM wind direction are:
```
ptend%u(i) = crm_output%ultend(i) * cos( -1.*crm_angle(i) ) + crm_output%vltend(i) * sin( -1.*crm_angle(i) )
ptend%v(i) = crm_output%vltend(i) * cos( -1.*crm_angle(i) ) - crm_output%ultend(i) * sin( -1.*crm_angle(i) )
```
I think these two should match

Reply:
These two should be equivalent, since 
```
cos(-x) == cos(x)
sin(-x) == -sin(x)
```